sdk/servicebus/ci.yml

```diff
@@ -33,3 +33,5 @@ extends:
       safeName: azuremgmtservicebus
     - name: azure_servicebus
       safeName: azureservicebus
+    SkipPythonVersion: 'pypy3'
```

Comment:
Should we make is a little more general and list the name of the default matrix entry to skip?

Reply:
Reason for not doing that is to avoid depending on a matrix name declared in a sub template. It will be fine now if we set   to name but if this option needs to be extended for live tests for a different py version then we need to ensure both pipeline's subset template follows same matrix naming. Another reason to ensure we skip the version all together irrespective of platforms we run on if same version is running on multiple platforms. For e.g. if we need to disable py 2.7 in the future for any specific service.
Please let me know if you think setting the matrix name is better. I will make this change.